to-do-app/src/components/TodoItem.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import IconCross from "../images/icon-cross.svg"
+import IconCheck from "../images/icon-check.svg"
+
+const TodoItem = (props) => {
+
```

Comment:
To avoid errors in a large project and to make the code more readable for other programmers, do the following convention:
const TodoItem = ( { content, isComplete, ...} ) ....
in that way the next programmer can know what the function needs when he put mouse over.
like here:
![image](https://user-images.githubusercontent.com/93907409/212486491-ad7e4f4e-ccf9-495e-ba38-a58a85899bba.png)

Reply:
nice, I did not know it is an option and now I do 
fixed it, thanks for this one👍